service/history/queues/queuestest/executable_dlq_test_suite.go

```diff
@@ -75,7 +76,9 @@ func TestExecutable(t *testing.T, tqm persistence.HistoryTaskQueueManager) {
 			clusterName := "test-cluster-" + t.Name()
 
 			var executable queues.Executable = NewFakeExecutable(task, tc.err)
-			executable = queues.NewExecutableDLQ(executable, tqm, clock.NewEventTimeSource(), clusterName)
+			ts := clock.NewEventTimeSource()
+			ts.Update(time.Now())
```

Comment:
nit: Add a small comment about why we need this here

Reply:
Done